integration/util/jest.js

```diff
@@ -97,7 +97,10 @@ expect.extend({
   toEthRevert(actual, reason = null) {
     if (reason === null) {
       return {
-        pass: !!actual['message'] && actual.message.startsWith(`Transaction has been reverted by the EVM:`),
+        pass: !!actual['message'] && (
+          actual.message.startsWith(`Transaction has been reverted by the EVM:`)
```

Comment:
These messages change on me daily-- I'm not sure why but whatever, they both mean the same thing